Orange/clustering/kmeans.py

```diff
@@ -27,15 +28,20 @@ class KMeans(Clustering):
 
     def __init__(self, n_clusters=8, init='k-means++', n_init=10, max_iter=300,
                  tol=0.0001, random_state=None, preprocessors=None,
-                 compute_silhouette_score=None):
+                 compute_silhouette_score=None, wrap_dask_ml=False):
```

Comment:
Ideally, the same class would handle Table and DaskTable without any new parameters, but sometimes this is hard.

I think we should avoid adding parameters for dask. Perhaps a subclass could be a better solution in this case?

I think we could also check whether to go for a solution Orange often uses when same methods can handle both classification and regression (which are seperate in scikit-lean). We often have a "main" class which then calls separate "worker" classes as needed, depending on whether it is given a classification or regression problem. Could you please  study how `modelling.RandomForestsLearner` works, just to get a feel for it?

I think that the first step to a solution would be to make a subclass for Dask and then we could perhaps check how to join them.